### ISSUE TITLE
Add config object

### ISSUE THREAD

**Issue by eddyerburgh** [vuejs/vue-test-utils#88]
I think we should add a config object.

We could use it to store mount options, and other options at a later date.

The main use I see right now is to set transition and transition-group Components to be stubbed by default.

This means the global default config would look like this:

```js
console.log(vueTestUtils.config) // {
  stubs: {
    'transition': TransitionStub,
    'transition-group': TransitionGroupStub
  }
}
```

The options in config would be overwritten by mount options passed to Wrapper.

Having a config would save adding large mount options in tests.

What are peoples thoughts?

**Comment by petridw** [comment]
Is there currently a way to have stubs apply to child components of the originally mounted component?

e.g. `mount(ComponentWithChildren, { stubs: { transition: TransitionStub } }` where the transition stub would also be applied recursively to any child components of `ComponentWithChildren`.

Seems having this global config option would be one option for enabling this behavior.

**Comment by eddyerburgh** [comment]
There isn't currently a way for doing this
 We could add it ☺

**Comment by eddyerburgh** [comment]
Closed with 7962b88f7535ce8f278b35b0ead1dcb6c2c0981c
